src/components/molecules/Progress/Progress.tsx

```diff
@@ -106,11 +116,13 @@ const Progress = ({ totalSteps, currentStep, withStep = false, progressColor, ..
 
   return (
     <ProgressWrapper>
+      <HiddenText id="in-progress">In progress</HiddenText>
```

Comment:
Might want to make this overridable or it'll be quite easy to have multiple ids which are the same in the DOM.

Reply:
we shouldn't have more than 1 progress bar on the page, no?

Reply:
Sorry to hijack this thread, can someone approve this https://github.com/ZopaPublic/react-components/pull/936/files :D

Reply:
In order to get out from being a maintainer you need to help us fix the checks :)